README.md

```diff
@@ -8,435 +8,429 @@ ACL (Access Control List)
+###Beginning with acl
 
-You cannot specify inherited ACEs in a manifest; you can only specify whether to allow upstream inheritance to flow into the managed target location (known as security descriptor). Please ensure your modeled resources follow this order or Windows will complain. The `acl` type **does not** enforce or complain about ACE order.
+For a basic implementation of the acl module, provide a target ACL and at least one permission:
 
-For a basic implementation of the acl module, you need to provide the target and at least one permission (access control entry or ACE). It will default the other settings to sensible defaults.
 
-    acl { 'c:/tempperms':
-      permissions => [
-       { identity => 'Administrator', rights => ['full'] },
-       { identity => 'Users', rights => ['read','execute'] }
-     ],
-    }
+~~~
+acl { 'c:/tempperms':
+  permissions => [
+   { identity => 'Administrator', rights => ['full'] },
+   { identity => 'Users', rights => ['read','execute'] }
+ ],
+}
+~~~
 
 ##Usage
 
-###Defined Type: acl
-
-The `acl` type contains two means of customization: [parameters](http://docs.puppetlabs.com/guides/custom_types.html#parameters) and [properties](http://docs.puppetlabs.com/guides/custom_types.html#properties). Parameters guide how the module operates. Properties guide ACL itself.
-
-Below is an example of the `acl` type with all parameters and properties.
-
-    acl { 'name':
-      target => 'absolute/path',
-      target_type => '<file>',
-      purge => '<true| false | listed_permissions>',
-      permissions => [
-        { identity => '<identity>',
-          rights => [<rights>],
-          perm_type => '<perm_type>',
-          affects => '<affects>',
-          child_types => '<child_types>'
-        }
-        ],
-      owner => '<owner>',
-      group => '<group>',
-      inherit_parent_permissions => '<true | false>',
-    }
 
-####Parameters
+A typical ACL is made up of access control entries (ACEs), which represent individual permissions. Each ACE comprises a defined trustee (an identity, representing a user, group, or system process), a set of rights, an inheritance and propagation strategy, and an allowed/denied status.
 
-#####`name`
+Each ACE has one of four possible statuses: 'explicit deny', 'explicit allow', 'inherited deny', and 'inherited allow'. The order of ACEs within the ACL determines which one is applied first.
 
-The name of the ACL resource; will be used for `target` if `target` is not set.
+You cannot specify inherited ACEs in a manifest; you can only specify whether to allow upstream inheritance to flow into the managed ACL. If your modeled resources don't follow this order, Windows complains. The `acl` type **does not** enforce or complain about ACE order.
```

Comment:
One thing here, the acl type will order them as they are put into the manifest, so it does enforce the order specified, it just doesn't complain about it being wrong. However you want to wordsmith that.